src/views/research/research.jsx

```diff
@@ -11,20 +11,26 @@ const Page = require('../../components/page/www/page.jsx');
 const Research = props => (
     <InformationPage title={props.intl.formatMessage({id: 'research.title'})}>
         <div className="inner info-inner">
-            <p><FormattedMessage id="research.conductors" /></p>
+            <p><FormattedMessage
+                id="research.conductors"
+                values={{
+                    ethicsCodeLink: (
+                        <a href="ETHICS CODE LINK">
+                            <FormattedMessage id="research.ethicsCodeLinkText" />
+                        </a>
+                    ),
+                    contactLink: (
+                        <a href="/contact-us/">
+                            <FormattedMessage id="research.contactLinkText" />
+                        </a>
+                    )
+                }}
+            /></p>
             <p>
                 <FormattedMessage
                     id="research.privacy"
```

Comment:
And then this id will need to be changed (to "research.collaborators" or whatever you decide to go with)